Makefile

```diff
@@ -134,7 +133,7 @@ endif
 test:
 	cd examples && go test -v -tags=all -parallel $(TESTPARALLELISM) -timeout 2h
 
-tfgen: install_plugins upstream docs
+tfgen: install_plugins docs
```

Comment:
@iwahbe do you know why upstream was removed from the docs? it looks like this is causing lint & build failure

cc @guineveresaenger

Reply:
resolved in a huddle